### ISSUE TITLE
How to use dexie in react hook

### ISSUE THREAD

**Issue by superhuai** [dfahlander/Dexie.js#1235]
i am according to the doc https://dexie.org/docs/dexie-react-hooks/useLiveQuery()

but, when i run,

it‘s wrong

```
./node_modules/dexie-react-hooks/dist/dexie-react-hooks.mjs
Can't import the named export 'liveQuery' from non EcmaScript module (only default export is 
available)
```

env: cra

**Comment by dfahlander** [comment]
Ok. What build system are you using? Webpack? Both dexie and dexie-react-hooks have ES module versions available so it should work. Which version of dexie?

**Comment by superhuai** [comment]
win10 

webpack 4.44.2

"dexie": "3.1.0-alpha.8",
"dexie-react-hooks": "^1.0.5",

**Comment by superhuai** [comment]
import {xxx} from 'dexie-react-hooks/dist/dexie-react-hooks.js'

it’s ok

**Comment by dfahlander** [comment]
ok, thanks for sharing. Seems to be a webpack issue but i couldn't be certain  If so I might need to remove the module entry from package.json in dexie-react-hooks to not confuse webpack. Strange that doesn't find dexie's module version.

**Comment by dfahlander** [comment]
As a workaround, I removed the module entry from package.json and released dexie-react-hooks@1.0.6 for now.

**Comment by superhuai** [comment]
thank you very much.
